### PR TITLE
Added Sign Up Screen

### LINKED ISSUE #2: Add Sign Up Screen

### PR DESCRIPTION
**Pull request by Samdid**
Closes #2 
Screenshot_1615456746

### CODE REVIEW

**Comment by Samdid** [comment]
@pratyushmp  Please review and merge this PR

File: lib/sign_up.dart [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            padding: const EdgeInsets.only(left:20.0,right: 20.0,bottom: 22.0),
+            child: Container(
+              height: MediaQuery.of(context).size.height*0.065,
+              child: TextFormField(
+                controller: _emailController,
+                onChanged: (value){
+
+                },
+                decoration: InputDecoration(
+                  labelText: 'Type something...',
```
Comment (infiniteoverflow) [inline lib/sign_up.dart:90]: Can you please remove the labelText , it wont be useful as you have already given a header for the respective textformfield.
Similarily , kindly update the remaining two TextFormFields in the same way.

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

File: lib/sign_up.dart (hunk 1 above)
Comment (Samdid) [reply lib/sign_up.dart:90]: Sure

**Comment by Samdid** [comment]
@infiniteoverflow Please check, I removed all three label text

**Pull request merged**